src/Jellyfin.Extensions/EnumerableExtensions.cs

```diff
@@ -1,51 +1,61 @@
     /// </summary>
-    public static class EnumerableExtensions
+    /// <param name="source">An instance of the <see cref="IEnumerable{String}"/> interface.</param>
+    /// <param name="value">The value to look for in the collection.</param>
+    /// <param name="stringComparison">The string comparison.</param>
+    /// <returns>A value indicating whether the value is contained in the collection.</returns>
+    /// <exception cref="ArgumentNullException">The source is null.</exception>
+    public static bool Contains(this IEnumerable<string> source, ReadOnlySpan<char> value, StringComparison stringComparison)
     {
-        /// <summary>
-        /// Determines whether the value is contained in the source collection.
-        /// </summary>
-        /// <param name="source">An instance of the <see cref="IEnumerable{String}"/> interface.</param>
-        /// <param name="value">The value to look for in the collection.</param>
-        /// <param name="stringComparison">The string comparison.</param>
-        /// <returns>A value indicating whether the value is contained in the collection.</returns>
-        /// <exception cref="ArgumentNullException">The source is null.</exception>
-        public static bool Contains(this IEnumerable<string> source, ReadOnlySpan<char> value, StringComparison stringComparison)
+        if (source == null)
         {
-            if (source == null)
-            {
-                throw new ArgumentNullException(nameof(source));
-            }
-
-            if (source is IList<string> list)
-            {
-                int len = list.Count;
-                for (int i = 0; i < len; i++)
-                {
-                    if (value.Equals(list[i], stringComparison))
-                    {
-                        return true;
-                    }
-                }
-
-                return false;
-            }
+            throw new ArgumentNullException(nameof(source));
+        }
 
-            foreach (string element in source)
+        if (source is IList<string> list)
+        {
+            int len = list.Count;
+            for (int i = 0; i < len; i++)
             {
-                if (value.Equals(element, stringComparison))
+                if (value.Equals(list[i], stringComparison))
                 {
                     return true;
                 }
             }
 
             return false;
         }
+
+        foreach (string element in source)
+        {
+            if (value.Equals(element, stringComparison))
+            {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /// <summary>
+    /// Gets an IEnumerable from a single item.
+    /// </summary>
+    /// <param name="item">The item to return.</param>
+    /// <typeparam name="T">The type of item.</typeparam>
+    /// <returns>The IEnumerable{T}.</returns>
+    public static IEnumerable<T> SingleItemAsEnumerable<T>(this T item)
```

Comment:
Not sure about this, creating an array with the one element may be more efficient

Reply:
I don't like it even if it's more efficient